UnityProject/Assets/Scripts/Messages/Server/UpdateMind.cs

```diff
@@ -0,0 +1,54 @@
+using System.Collections;
+using System.Collections.Generic;
+using Messages.Server;
+using Mirror;
+using UnityEngine;
+
+public class UpdateMind : ServerMessage<UpdateMind.NetMessage>
+{
+	public struct NetMessage : NetworkMessage
+	{
+		public uint Mind;
+	}
+	public override void Process(NetMessage msg)
+	{
+		LoadNetworkObject(msg.Mind);
+		if (NetworkObject == null)
+		{
+			PlayerManager.SetMind(null);
+		}
+
+		if (NetworkObject.TryGetComponent<Mind>(out var Mind))
```

Comment:
if you are going to set the result of getting the component regardless, why don't you just use the regular ``GetComponent<T>``?